src/i18n.c

```diff
@@ -32,6 +33,10 @@ i18n_info i18n_get_info(char *system_name){
         }
     }
     char *argv_title[] = {title, system_name};
+    if (!strncmp(lang, "ja_JP", 5)) {
```

Comment:
Spliting `title` into `title_pre` & `title_suf` may be better.

Reply:
> Spliting `title` into `title_pre` & `title_suf` may be better.

OK. I did that in the commit `1c44d63`.